gradle.properties

```diff
@@ -1 +1 @@
-version=3.0.3
```

Comment:
use the shell script instead - it automatically updates this file and all the others

Reply:
I've tried now - there don't seem to be any new changes

Reply:
it updates readme files :D. for library - it matters